### ISSUE TITLE
Type-o in module

### ISSUE THREAD

**Issue by xorander00** [ray-x/navigator.lua#218]
Great project! :)  
    
Just noticed a type-o here while looking at the mappings and wonder why formatting wasn't working...  
https://github.com/ray-x/navigator.lua/blob/master/lua/navigator/formatting.lua#L34  
  
Also, is this a type-o too or is this what it's supposed to be? It's binding in normal mode but uses a _v function?  
https://github.com/ray-x/navigator.lua/blob/master/lua/navigator/lspclient/mapping.lua#L64  
  
I'd fork and create a PR, but I'm pressed for time currently and decided to post this instead since it's so small.

**Comment by ray-x** [comment]
You are correct, there was a typo.
Also the range_format keymap only works if the lsp support it. AFAIK, most lsp does not support this feature.

**Comment by avently** [comment]
```
Error executing vim.schedule lua callback: vim/_init_packages.lua:0: .../start/navigator.lua/lua/navigator/lspclient/mapping.lua:392: 'end' expected (to close 'function' at line 131) near '<eof>'
stack traceback:
        [C]: in function 'error'
        vim/_init_packages.lua: in function <vim/_init_packages.lua:0>
        [C]: in function 'require'
        ...r/start/navigator.lua/lua/navigator/lspclient/attach.lua:33: in function 'on_attach'
        .../start/navigator.lua/lua/navigator/lspclient/clients.lua:396: in function '_on_attach'
        ...ck/packer/start/nvim-lspconfig/lua/lspconfig/configs.lua:280: in function '_setup_buffer'
        ...ck/packer/start/nvim-lspconfig/lua/lspconfig/configs.lua:183: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by ray-x** [comment]
thanks, should been fixed #219
